python/lsst/ts/dcs/dcs_csc.py

```diff
@@ -0,0 +1,55 @@
+# import asyncio
+import pathlib
+from lsst.ts import salobj
+
+__all__ = ['DCSCSC']
```

Comment:
Consider renaming the class to `DCS` or `CSC`.

Reply:
I suggest naming the CSC for the SAL component name (e.g. ATDomeCsc). Is there going to be a DSC SAL component? I see Dome but not DSC in the current XML. If you are creating a new SAL component then please consider giving it a less cryptic name. I'm not fond of three-letter acronyms and we already have DSM, which could all-too-easily be confused with DSC.

Also if you spell the end of it Csc instead of CSC then DSMCsc should not be too bad (if you are stuck with DSM as the SAL component name).

For what it's worth, my code normally follows the DM naming convention: CamelCase for class names, even for acronyms. Thus BaseCsc, ConfigurableCsc, TestCsc. etc. But I deviate this when needed to match SAL component names. Thus ATDome instead of AtDome.

Reply:
Thanks for the suggestions and explanations. I renamed it to DomeCsc.